src/components/IncidentTable/IncidentTableComponent.scss

```diff
@@ -103,6 +103,15 @@
   position: relative;
 }
 
+/* Severity Badge */
```

Comment:
Added new CSS class due to overrunning text for some i18n tokens on the severity cell.